static/js/portico/landing-page.js

```diff
@@ -220,6 +220,55 @@ var events = function () {
     }
 };
 
+var billingEvents = function (e) {
+    $("body").on("click", ".stripe", function(e) {
+        const { key, image, name, label, description } = this.dataset;
+
+        var handler = StripeCheckout.configure({
+            key,
+            image,
+            locale: 'auto',
+            name,
```

Comment:
I think you should include `email` here as well so that we can identify who added the card.

Reply:
Updated.